src/stylesheets/elements/_inputs.scss

```diff
@@ -261,6 +261,14 @@ legend {
   cursor: not-allowed;
 }
 
+@media print {
+  [type=checkbox]:checked + label::before,
+  [type=checkbox]:checked:disabled + label::before {
+    content: url('#{$image-path}/correct8-alt.png');
+    content: url('#{$image-path}/correct8-alt.svg');
```

Comment:
- Add an explicit background-color of `white` here, just in case the user decides to print background images. 
- It might also be worth explicitly setting the background image to `none` as well. That could make this solution a bit more portable if we wanted to apply something similar elsewhere.
- Set `text-indent: 0` to center the new image in the checkbox